test/blackbox-tests/test-cases/directory-targets/main.t

```diff
@@ -108,7 +108,7 @@ Hints for directory targets.
 
   $ dune build outputs
   Error: Don't know how to build outputs
-  Hint: did you mean output?
+  Hint: did you mean output/?
```

Comment:
I think it looks a bit confusing. Firstly, we don't support the syntax `output/` to specify directory targets from CLI (I believe?). Secondly, even if we supported it, it's not obvious what this syntax stands for.

I think, we should just treat directory targets in a special way, and maybe have a separate hint for them, e.g. `did you mean to build a directory target "output"`?